src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -122,7 +122,10 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     executionResult match {
       case SuccessfulExecution(outputs, returnCode) => context.parent ! WorkflowActor.CallCompleted(key, outputs, returnCode)
       case AbortedExecution => context.parent ! WorkflowActor.AbortComplete(key)
-      case FailedExecution(e, returnCode) => context.parent ! WorkflowActor.CallFailed(key, returnCode, e.getMessage)
+      case FailedExecution(e, returnCode) =>
+        logger.error("Failing call: " + e.getMessage, e)
+        log.error(e, e.getMessage)
```

Comment:
I added the second call to the other logger because at least in console mode the first call to `WorkflowLogger` didn't print the stack trace (which was the point of passing the `Throwable`.  I was also a bit surprised that with "run server" Cromwell defaults to console mode instead of server mode.